client/templates/fulfillmentOrders/fulfillmentOrders.js

```diff
@@ -82,32 +81,34 @@ function context(routeName) {
 }
 
 Template.fulfillmentOrders.onCreated(function () {
-  let instance = this;
-  instance.autorun(function () {
-    let currentRoute = Router.current().route.getName();
+  this.autorun(() => {
+    let currentRoute = ReactionRouter.getRouteName();
     let result = context(currentRoute);
-    let params = Router.current().params.status || Router.current().params.date;
+    let params = ReactionRouter.getParam('status') || ReactionRouter.getParam('date');
 
     if (params) {
-      instance.subscribe(result.subscription, params);
+      this.subscribe(result.subscription, params);
     } else {
-      instance.subscribe(result.subscription);
+      this.subscribe(result.subscription);
     }
   });
-  Session.set('selectedOrders', []);
+  Session.setDefault('selectedOrders', []);
```

Comment:
I think session default just sets a default if it's cleared out. So if you did `Session.remove()` that it would still be an array rather than undefined? But that is a valid concern, I have never though but I think we use different sessions on other options?